apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/SessionData.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Cloud.BigQuery.V2
+{
+    /// <summary>
+    /// Keep constants for session's temporary data.
+    /// </summary>
+    public static class SessionData
+    {
+        /// <summary>
+        /// It is used to refer the temporary dataset id in sessions.
```

Comment:
Just "The ID of the temporary dataset within a session."

(In general, any summary starting with "It is" can be improved...)